content/en/docs/setup/independent/high-availability.md

```diff
@@ -447,7 +447,7 @@ Only follow this step if your etcd is hosted on dedicated nodes (**Option 1**).
 ## Run kubeadm init on master0 {#kubeadm-init-master0}
 
 1.  In order for kubeadm to run, you first need to write a configuration file:
-      ```none
+      ```bash
```

Comment:
This is based purely on a hunch; i.e., the code block defined above [here](https://github.com/kubernetes/website/pull/9020/files#diff-b9cfd9e8dc83e82d25ccb8ead002e754R440) renders correctly.